docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md

```diff
@@ -127,6 +128,65 @@ Enable the following behaviors by registering the plugins:
+| MerchantProductProductAbstractPostCreatePlugin | Creates a new merchant product abstract entity if `ProductAbstractTransfer.idMerchant` is set. | None | Spryker\Zed\MerchantProduct\Communication\Plugin\Product |
+| PriceProductAbstractPostCreatePlugin | Creates new product price entities by abstract product id and price type if they don't exist. | None | Spryker\Zed\PriceProduct\Communication\Plugin\Product |
+| PriceProductProductAbstractExpanderPlugin | Expands product abstract transfer with prices. | None | Spryker\Zed\PriceProduct\Communication\Plugin\Product |
+| ImageSetProductAbstractPostCreatePlugin | Persists all provided image sets to database for the given abstract product. | None | Spryker\Zed\ProductImage\Communication\Plugin\Product |
+| ProductImageProductAbstractExpanderPlugin | Expands product abstract transfer with product images. | None | Spryker\Zed\ProductImage\Communication\Plugin\Product |
+| TaxSetProductAbstractExpanderPlugin | Finds tax set in database by `ProductAbstractTransfer.idProductAbstract` and sets ProductAbstractTransfer.idTaxSet transfer property. | None | Spryker\Zed\TaxProductConnector\Communication\Plugin\Product |
+| TaxSetProductAbstractPostCreatePlugin | Saves tax set id to product abstract table. | None | Spryker\Zed\TaxProductConnector\Communication\Plugin\Product |
+
+**src/Pyz/Zed/Product/ProductDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Product;
+
+use Spryker\Zed\MerchantProduct\Communication\Plugin\Product\MerchantProductProductAbstractPostCreatePlugin;
+use Spryker\Zed\PriceProduct\Communication\Plugin\Product\PriceProductAbstractPostCreatePlugin;
+use Spryker\Zed\PriceProduct\Communication\Plugin\Product\PriceProductProductAbstractExpanderPlugin;
+use Spryker\Zed\ProductImage\Communication\Plugin\Product\ImageSetProductAbstractPostCreatePlugin;
+use Spryker\Zed\ProductImage\Communication\Plugin\Product\ProductImageProductAbstractExpanderPlugin;
+use Spryker\Zed\TaxProductConnector\Communication\Plugin\Product\TaxSetProductAbstractExpanderPlugin;
+use Spryker\Zed\TaxProductConnector\Communication\Plugin\Product\TaxSetProductAbstractPostCreatePlugin;
+use Spryker\Zed\Product\ProductDependencyProvider as SprykerProductDependencyProvider;
+
+class ProductDependencyProvider extends SprykerProductDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\ProductExtension\Dependency\Plugin\ProductAbstractPostCreatePluginInterface[]
+     */
+    protected function getProductAbstractPostCreatePlugins(): array
+    {
+        return [
+            new ImageSetProductAbstractPostCreatePlugin(),
+            new TaxSetProductAbstractPostCreatePlugin(),
+            new PriceProductAbstractPostCreatePlugin(),
+            new MerchantProductProductAbstractPostCreatePlugin(),
```

Comment:
as I mentioned in previous comment only `MerchantProductProductAbstractPostCreatePlugin` should be mentioned in MP IG since other plugins are part of `Product` feature